src/validate-title.ts

```diff
@@ -15,7 +15,7 @@ export default function isValidTitle(title: string): ValidTitle {
     mergeCorrespondence: ['id', 'source'],
 
     // allow comma and slash in scope
-    headerPattern: /^(\w*)(?:\(([\w$.\-*,/ ]*)\))?: (.*)$/
+    headerPattern: /^(\w*)(?:\(([\w$.\-*,/ ]*)\))?!?: (.*)$/
```

Comment:
I don't like how the conventional commits parser [doesn't default to allowing `!` in titles](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-commits-parser#headerpattern)...